logger/src/lib.rs

```diff
@@ -676,20 +675,27 @@ impl Logger {
 
         set_max_level(Level::Trace.to_level_filter());
 
+        self.log_helper(
+            format!("Running {} v{}", app_info.name, app_info.version),
+            Some(Destination::Pipe),
+        );
+        LOGGER.level_info.set_writer(Destination::Pipe);
```

Comment:
If this statement switched places with the one above, would the `forced_destination` still be necessary? The way I see it, the destination would be set to pipe, then `log_helper` should pick it up.

Reply:
I guess it would still be necessary because there may still be an async request that is executed exactly between the moment when `LOGGER.level_info.set_writer(Destination::Pipe);` finished and `self.log_helper` begins resulting in the same issue as before.

It would be a lot less likely but still possible.